packages/client/src/forms/index.ts

```diff
@@ -1228,6 +1238,7 @@ export function fieldTypeLabel(type: IFormField['type']) {
     RADIO_GROUP_WITH_NESTED_FIELDS: messages.radioGroupWithNestedField,
     INFORMATIVE_RADIO_GROUP: messages.informativeRadioGroup,
     CHECKBOX_GROUP: messages.checkboxGroup,
+    CHECKBOX: messages.checkbox,
```

Comment:
You will need to add the `checkbox` message that you are trying to reference here

Reply:
Alright